### ISSUE TITLE
Spelling and grammatical errors

### ISSUE THREAD

**Issue by michael-coker** [syndesisio/syndesis#1519]
Noticed some inconsistencies in punctuation, grammar, case, etc. 

@TovaCohen would you mind reviewing these? Some of these are in the doc I shared with you a while back and some are new.

> /integrations/create/action-select/0 (twitter)
* "Search for tweets that mention you" - needs period
* "Search tweets based one or more keywords" - needs period

> /integrations/create/action-select/0 (database)
* "Invoke SQL to periodically get data out"
* "Periodically Invoke a Stored Procedure to get Data out" - uncapitalize "Invoke" and "Data" and add period

> /integrations/create/action-configure/0/0 (database - periodic SQL invocation)
* "Enter a SQL Statement starting with SELECT" - needs period

> /integrations/create/action-configure/0/0 (database - periodic stored prodecure)
* "Select the Stored Procedure" - needs period

> /integrations/create/action-select/0 (salesforce)
* All of the descriptions say "when an Salesforce record is..." - change "an" to "a"

> /integrations/create/action-configure/0/0 (salesforce)
* "Specify the Salesforce object name for which to receive notification when records are created"

> /integrations/create/action-select/1 (salesforce - create or update record)
* All of the descriptions after clicking "Create or update record," "Delete record," etc. need a period.
* "Account, Contact or Order" - needs comma

> /integrations/create/action-configure/1/0 (salesforce)
* "Specify the Salesforce object name of the record to create" - needs period

> /integrations/create/step-select/1
* "Map fields from the input type to the output type" - needs period

> /customizations/api-connector/create/swagger-connector
* "Custom API's are RESTful APIs" - remove single quote
* "as long as a well-documented swagger is available and conforms to OpenAPI standard." - say "to OpenAPI standards" or "to the OpenAPI standards"

> /settings/oauth-clients
* is the "OAuth Client Management" tab necessary since there are no other tabs?
  * If so, should that text match the <h1> on the page, or vice versa?
* When you save settings:
  * "You can validate these values when creating a connection Salesforce" - wording and needs period
  * "You can validate these values when creating a connection Twitter" - wording and needs period

> /customizations/extensions (empty state)
* "There are no extensions available" - needs period

> /integrations
* When you publish or unpublish an integration from the list, the toast notification needs punctuation between the bold and normal text.
* "Integration is activating Please allow a moment for the integration to fully activate."
* "Integration is deactivating Please allow a moment for the integration to be unpublished."

**Comment by TovaCohen** [comment]
I ADDED COMMENTS:

    /integrations/create/action-select/0 (twitter)

    "Search for tweets that mention you" - needs period  
YES
    "Search tweets based one or more keywords" - needs period 
YES AND IT IS MISSING A WORD. I SUGGEST THIS BE CHANGED TO: Search for tweets that contain text that you specify.

    /integrations/create/action-select/0 (database)

    "Invoke SQL to periodically get data out"
    "Periodically Invoke a Stored Procedure to get Data out" - uncapitalize "Invoke" and "Data" and add period
YES BUT I SUGGEST REWORDING THESE TO:
Periodically invoke SQL to  obtain, store, update, or delete data. 
Periodically invoke a stored procedure.

    /integrations/create/action-configure/0/0 (database - periodic SQL invocation)

    "Enter a SQL Statement starting with SELECT" - needs period
YES, BUT I'M SEEING A LONGER SENTENCE THAT SHOULD BE:
Enter a SQL statement that starts with INSERT, SELECT, UPDATE, or DELETE.

    /integrations/create/action-configure/0/0 (database - periodic stored prodecure)

    "Select the Stored Procedure" - needs period
YES AND MAKE IT LOWERCASE:
Select the stored procedure.

    /integrations/create/action-select/0 (salesforce)

    All of the descriptions say "when an Salesforce record is..." - change "an" to "a" 
YES

    /integrations/create/action-configure/0/0 (salesforce)

    "Specify the Salesforce object name for which to receive notification when records are created"
THAT NEEDS A PERIOD AT THE END.
IT LOOKS LIKE AFTER YOU SELECT A SALESFORCE ACTION, THE NEXT PAGE HAS A SENTENCE AT THE TOP THAT JUST ABOUT ALWAYS NEEDS A PERIOD AT THE END. 

    /integrations/create/action-select/1 (salesforce - create or update record)

    All of the descriptions after clicking "Create or update record," "Delete record," etc. need a period.
AGREED
    "Account, Contact or Order" - needs comma
RIGHT AND I WOULD REVISE TO:
Create a new Salesforce record, such as an Account, Contact, or Order.

    /integrations/create/action-configure/1/0 (salesforce)

    "Specify the Salesforce object name of the record to create" - needs period
YES

    /integrations/create/step-select/1

    "Map fields from the input type to the output type" - needs period
YES

    /customizations/api-connector/create/swagger-connector

    "Custom API's are RESTful APIs" - remove single quote
RIGHT AND I WOULD REWORD:
Provide a Swagger specification that describes a REST API. The API can be hosted anywhere. The Swagger specification must conform to the OpenAPI (the new name for Swagger) specification.

    "as long as a well-documented swagger is available and conforms to OpenAPI standard." - say "to OpenAPI standards" or "to the OpenAPI standards"

    /settings/oauth-clients

    is the "OAuth Client Management" tab necessary since there are no other tabs?
        If so, should that text match the
        on the page, or vice versa?
THERE IS AN ISSUE ABOUT THIS SOMEWHERE. I BROUGHT THIS UP BEFORE.
THEY SHOULD BE THE SAME. AND I THINK BOTH SHOULD BE
OAuth Application Management
IN THIS CONTEXT, SYNDESIS IS A CLIENT OF THE APPLICATIONS THAT SYNDESIS IS REGISTERING WITH. 

    When you save settings:
        "You can validate these values when creating a connection Salesforce" - wording and needs period
        "You can validate these values when creating a connection Twitter" - wording and needs period
YES. TO FIX THE WORDING, INSERT "to" right after the link and before the name of the application.
So you have:
You can validate these values when creating a connection to Salesforce.

    /customizations/extensions (empty state)

    "There are no extensions available" - needs period
YES

    /integrations

    When you publish or unpublish an integration from the list, the toast notification needs punctuation between the bold and normal text.
    "Integration is activating Please allow a moment for the integration to fully activate."
    "Integration is deactivating Please allow a moment for the integration to be unpublished."
AGREED. INSERT A PERIOD AFTER THE BOLD PART.
